project_stage_state/models/project_task_type.py

```diff
@@ -0,0 +1,20 @@
+# Daniel Reis, 2014
```

Comment:
Copyright should be be as per https://github.com/OCA/maintainer-tools/blob/d9e2c3b8dc163c8d0fcdd36278b782074c3cff12/template/module/__manifest__.py#L2

Reply:
@alexey-pelykh  , Could you please check it now?